### ISSUE TITLE
println does not print nil slices

### ISSUE THREAD

**Issue by thehowl** [gnolang/gno#1377]
```go
package main

func main() {
        var a []string
        println(a)
}
```

panics with:

```
panic: interface conversion: gnolang.Value is nil, not *gnolang.SliceValue [recovered]
        panic: interface conversion: gnolang.Value is nil, not *gnolang.SliceValue

goroutine 1 [running]:
main.runExpr.func1()
        /home/howl/oc/gno2/gnovm/cmd/gno/run.go:158 +0x125
panic({0xc0ae00?, 0xc00047c840?})
        /usr/lib/go/src/runtime/panic.go:914 +0x21f
github.com/gnolang/gno/gnovm/pkg/gnolang.(*TypedValue).Sprint(0xc0001218c8, 0xc00047c780?)
        /home/howl/oc/gno2/gnovm/pkg/gnolang/values_string.go:230 +0xe71
github.com/gnolang/gno/gnovm/pkg/gnolang.UverseNode.func11(0xc000177200)
        /home/howl/oc/gno2/gnovm/pkg/gnolang/uverse.go:947 +0x1ee
github.com/gnolang/gno/gnovm/pkg/gnolang.(*Machine).doOpCallNativeBody(...)
        /home/howl/oc/gno2/gnovm/pkg/gnolang/op_call.go:166
````

**Comment by notJoon** [comment]
The section in gnolang where println is called is from lines [935 to 955](https://github.com/gnolang/gno/blob/master/gnovm/pkg/gnolang/uverse.go#L935-L955) in uverse.go. It seems to be an issue caused by not handling nil values in this part. However, when the same code is run in Go, it outputs [0/0]0x0. Should we modify it to simply print `undefined` instead?

**Comment by thehowl** [comment]
> Should we modify it to simply print `undefined` instead?

Yeah, sounds adequate.

**Comment by ajnavarro** [comment]
@thehowl do we want to modify standard Go behavior on Gno changing `[0/0]0x0` output to `undefined`?

**Comment by thehowl** [comment]
@ajnavarro `println` and `print` are not properly "standardized" function where we should strive to have 100% compatibility with Go. See: [Go spec#Bootstrapping](https://go.dev/ref/spec#Bootstrapping), [builtin#println](https://pkg.go.dev/builtin#println). The documentation explicitly says:

>The println built-in function formats its arguments **in an implementation-specific way**

(emphasis mine)

I think printing `undefined` is acceptable as it is consistent with the behaviour when you perform `println(nil)`.

**Comment by thehowl** [comment]
Fixed by #1380

(@notJoon use "linking keywords" for your next PRs; see [github doc](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
